packages/client/src/views/Settings/ChangeEmailModal/ChangeEmailView.tsx

```diff
@@ -56,13 +58,23 @@ export function ChangeEmailView({ show, onSuccess, onClose }: IProps) {
   const restoreState = () => {
     setEmailAddress(EMPTY_STRING)
     setIsInvalidEmailAddress(false)
+    setUnknownError(false)
   }
   const toggleDuplicateEmailErrorNotification = () => {
     setShowDuplicateEmailErrorNotification((prevValue) => !prevValue)
   }
+  const toggleUnknownErrorNotification = () => {
+    setUnknownError((prevValue) => !prevValue)
+  }
   const continueButtonHandler = async (emailAddress: string) => {
-    const userData = await queriesForUser.fetchUserDetailsByEmail(emailAddress)
-    const emailExists = userData.data.getUserByEmail
+    let userData
+    try {
+      userData = await queriesForUser.fetchUserDetailsByEmail(emailAddress)
```

Comment:
https://www.apollographql.com/docs/react/data/queries/#manual-execution-with-uselazyquery

Could we utilize this pattern here to directly get the "error" from Apollo